.github/workflows/build.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           make test
   acceptance:
     needs: [go-version, build]
-    runs-on: custom-linux-large
+    runs-on: ubuntu-latest
```

Comment:
The difference between these two is a matter of minutes. So, it should not be a problem to use a less beefy machine.